must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallVisitor.java

```diff
@@ -25,6 +27,29 @@ public MustCallVisitor(BaseTypeChecker checker) {
     super(checker);
   }
 
+  @Override
+  protected boolean skipReceiverSubtypeCheck(
+      MethodInvocationTree node,
+      AnnotatedTypeMirror methodDefinitionReceiver,
+      AnnotatedTypeMirror methodCallReceiver) {
+    // TODO: Check explicit receiver parameters annotated with @Owning. ExecutableElement
```

Comment:
Does it work to get the annotations on the receiver type, rather than from the element?

Reply:
No. I did also try that :)

Reply:
Ok.  I think we're fine to just never allow ownership transfer for receivers for now